api/v1alpha3/awsmachine_types.go

```diff
@@ -36,6 +36,17 @@ type AWSMachineSpec struct {
 	// AMI is the reference to the AMI from which to create the machine instance.
 	AMI AWSResourceReference `json:"ami,omitempty"`
 
+	// ImageLookupFormat is the AMI naming format to look up machine images when
+	// a machine does not specify an AMI. When set, this will be used for all
+	// cluster machines unless a machine specifies a different ImageLookupOrg.
```

Comment:
This looks like copy-pasta from the cluster type to me.

```suggestion
	// ImageLookupFormat is the AMI naming format to look up the image for this machine
	// It will be ignored if an explicit AMI is set.
```

Reply:
Good idea. It makes more sense what is going on with the vars. Both items have been changed